deploy/docker/base/requirements.txt

```diff
@@ -1,4 +1,6 @@
 cryptography==3.4.6
 Flask==1.1.2
 PyJWT==1.7.1
-requests==2.22.0
\ No newline at end of file
+requests==2.22.0
+jaeger_client
```

Comment:
In order to avoid future library update problems, the best would be to set a fixed version.